platform/iphone/app_delegate.h

```diff
@@ -28,20 +28,17 @@
 /* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
 /*************************************************************************/
 
-#import "gl_view.h"
-#import "view_controller.h"
 #import <UIKit/UIKit.h>
-
+#import <Foundation/Foundation.h>
 #import <CoreMotion/CoreMotion.h>
 
-@interface AppDelegate : NSObject <UIApplicationDelegate, GLViewDelegate> {
-	//@property (strong, nonatomic) UIWindow *window;
-	ViewController *view_controller;
-	bool is_focus_out;
-};
+#import "gl_view.h"
+#import "view_controller.h"
```

Comment:
It was correct before, Godot's own headers should come first, then system headers (unless a different order is necessary for technical reasons). See http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes